docs/dokka-presets/assets/logo-icon.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="16px" height="16px" viewBox="0 0 16 16" enable-background="new 0 0 16 16" xml:space="preserve">  <image id="image0" width="16" height="16" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
+AAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAkFBMVEX////9mCf9mCf9mCf9
+mCf9mCf9mCf9mCf9mCfzfCP0fiP6jyb2hSXqYyDsaCH1gSTsaSHpXyD2hSTvcSLzfSP4iSXyeiPp
+YCDzfCPzfCPpXyDpXyDzfCPzfCPpXyDpXyDzfCPzfCPpXyDpXyDzfCPzeyPpXyDpXyD9mCfzfCP7
+kyb5jSbpXyD8lif8lCf///9jOjcpAAAAKHRSTlMAUOAwwBCggPBA8PX18Ezo6EDo6Pbw8PaA8PCA
+EKCgEDDAwDBQ6OBQjKuQ+AAAAAFiS0dEAIgFHUgAAAAHdElNRQflBBwRMSsSF1nDAAAAeElEQVQY
+053PyQKCMAxF0ZS5DqgIiiggOJBI8P8/TwuFsuau+s6qAegTliXAZDv4z7H1dD3Uea7afoBTgQ8g
+V2sDm20Ikmj3GWa7Zz4oIIo6xO+ReQKKk9OZ50DEvBwgvRjIruqrt3yEotTX3CsF9WN27/P1bobX
+D4qaG2DrJKPCAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIxLTA0LTI4VDE3OjQ5OjQzKzAzOjAwXlQE
+twAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMS0wNC0yOFQxNzo0OTo0MyswMzowMC8JvAsAAAAASUVO
+RK5CYII=" />
+</svg>
```

Comment:
**Question:** What is this logo used for? I don't see it referenced anywhere other than the Dokka custom asset list. If it's not used, can it be removed?

If it is needed, I notice that it's an SVG that contains only a 16x16 bitmap which renders poorly at non-default DPI. Could we maybe include an actual vector image (instead of a vector wrapper of a raster image)?

Reply:
I believe this icon is used as the "favicon".  Here is an example from generated docs:

```
services/dynamodb/dynamodb/aws.sdk.kotlin.services.dynamodb.model/-create-backup-request/to-string.html:</head><body><link href="../../../../../../../../images/logo-icon.svg" rel="icon" type="image/svg">
...
```

As such we should probably just use whatever other AWS SDKs use.  I'll take a peek.

Reply:
I spent a laughable (cryable?) amount of time on this but it turns out that the ico file is unused.  The svg specified in the build file is taken in by Dokka and used in place of the Kotlin logo.  From my experiments it does not seem that dokka cares about the ico file, nor did I find a way to specify that instead of an svg.  I don't know where the svg came from (@aajtodd ?) but it does seem to look like the AWS favicon.ico that is referenced internally.  I've asked internally for better assets and if I get an answer I'll update in a subsequent PR.